<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>message_qemu</class>
 <widget class="QWidget" name="message_qemu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>1902</width>
       <height>1062</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1902</width>
       <height>1062</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>1884</width>
          <height>260</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1884</width>
          <height>260</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>371</width>
             <height>241</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>371</width>
             <height>241</height>
            </size>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1189</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>291</width>
             <height>241</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>291</width>
             <height>241</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>1884</width>
          <height>777</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1884</width>
          <height>777</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QTabWidget" name="tabWidget">
           <property name="styleSheet">
            <string notr="true">QTabBar::tab{width:105}
QTabBar::tab{height:40}
QTabWidget::tab-bar
{
	background:transparent;
	subcontrol-position:center;
}
QTabWidget:pane {border-top:0px solid #e8f3f9;background:  transparent; }</string>
           </property>
           <property name="tabPosition">
            <enum>QTabWidget::West</enum>
           </property>
           <property name="tabShape">
            <enum>QTabWidget::Rounded</enum>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <property name="elideMode">
            <enum>Qt::ElideNone</enum>
           </property>
           <property name="usesScrollButtons">
            <bool>false</bool>
           </property>
           <property name="movable">
            <bool>true</bool>
           </property>
           <property name="tabBarAutoHide">
            <bool>false</bool>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>RTA</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="minimumSize">
                <size>
                 <width>600</width>
                 <height>400</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>600</width>
                 <height>400</height>
                </size>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_2">
            <attribute name="title">
             <string>Configure</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QTabWidget" name="tabWidget_2">
               <property name="styleSheet">
                <string notr="true">QTabBar::tab{width:105}
QTabBar::tab{height:40}
QTabWidget:pane {border-top:0px solid #e8f3f9;background:  transparent; }
QTabWidget::tab-bar
{
	background:transparent;
	subcontrol-position:left;
}</string>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <property name="iconSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
               <widget class="QWidget" name="tab_3">
                <attribute name="title">
                 <string>RTA</string>
                </attribute>
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="0" column="0">
                  <widget class="QPushButton" name="pushButton">
                   <property name="minimumSize">
                    <size>
                     <width>100</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Com1</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QPushButton" name="pushButton_2">
                   <property name="minimumSize">
                    <size>
                     <width>100</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">pushButton_2-&gt;setStyleSheet(&quot;background-color: rgba(0, 0, 0, 0)&quot;);</string>
                   </property>
                   <property name="text">
                    <string>Com2</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QPushButton" name="pushButton_3">
                   <property name="minimumSize">
                    <size>
                     <width>100</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Com3</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QPushButton" name="pushButton_6">
                   <property name="minimumSize">
                    <size>
                     <width>100</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Com4</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QPushButton" name="pushButton_4">
                   <property name="minimumSize">
                    <size>
                     <width>100</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Com5</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QPushButton" name="pushButton_5">
                   <property name="minimumSize">
                    <size>
                     <width>100</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Com6</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="tab_4">
                <attribute name="title">
                 <string>Configure</string>
                </attribute>
               </widget>
               <widget class="QWidget" name="tab_5">
                <attribute name="title">
                 <string>哈</string>
                </attribute>
               </widget>
               <widget class="QWidget" name="tab_6">
                <attribute name="title">
                 <string>嘿</string>
                </attribute>
               </widget>
               <widget class="QWidget" name="tab_7">
                <attribute name="title">
                 <string>吼</string>
                </attribute>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_8">
            <attribute name="title">
             <string>页</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tab_9">
            <attribute name="title">
             <string>页</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tab_10">
            <attribute name="title">
             <string>页</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tab_11">
            <attribute name="title">
             <string>页</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
